app/controllers/api/v2/contacts_controller.rb

```diff
@@ -0,0 +1,32 @@
+module API
+  module V2
+    class ContactsController < API::V2::ApplicationController
+      deserializable_resource :contact, class: API::V2::DeserializableContact
+
+      def update
+        authorize contact
+
+        if contact.update(contact_params)
+          render jsonapi: contact, status: :ok
+        else
+          render jsonapi_errors: contact.errors, status: :unprocessable_entity
```

Comment:
Can we have the unhappy path tests please.

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1579/commits/996cb2eaadcae1d8aa7cb9fa0755d5682656c18b